dotcom-rendering/src/web/lib/transparentColour.ts

```diff
@@ -0,0 +1,11 @@
+export const transparentColour = (
```

Comment:
To be honest I'm not completely following this code. What problem is this solving? Maybe I can use that to reason it through?

Reply:
I think this takes a hex string like #ABCDEF, transforms each tuple (AB, CD, EF) into its RGB equivalent and adds 50% transparency (default, unless overridden).

Needed because at the moment every headline is using `rgba(171, 6, 19, 0.5)` everywhere unless I misunderstood?

Reply:
Thanks @MarSavar – I will split it out in another PR though. And expands some of the JSDoc.